### PR TITLE
mubeng 0.14.2 (new formula)

### PR DESCRIPTION
**Pull request by dwisiswant0**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by chenrui333** [comment]
@dwisiswant0 can you update your PR to allow push access?

File: Formula/m/mubeng.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class Mubeng < Formula
+  desc "Incredibly fast proxy checker & IP rotator with ease"
+  homepage "https://github.com/kitabisa/mubeng"
+  url "https://github.com/kitabisa/mubeng/archive/refs/tags/v0.14.2.tar.gz"
+  sha256 "27842f0d587ced3b79b3c5e68be7e59272b0f2e89f754e3322d17bf7eda6802c"
+  license "Apache-2.0"
```
Comment (chenrui333) [inline Formula/m/mubeng.rb:6]:
```suggestion
  license "Apache-2.0"
  head "https://github.com/kitabisa/mubeng.git", branch: "master"
```

File: Formula/m/mubeng.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -X ktbs.dev/mubeng/common.Version=#{version}"), "./cmd/mubeng"
```
Comment (chenrui333) [inline Formula/m/mubeng.rb:11]:
```suggestion
    ldflags = "-s -w -X ktbs.dev/mubeng/common.Version=#{version}"
    system "go", "build", *std_go_args(ldflags:), "./cmd/mubeng"
```

File: Formula/m/mubeng.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  end
+
+  test do
+    assert_match "no proxy file provided", shell_output("#{bin}/mubeng")
+    assert_match version.to_s, shell_output("#{bin}/mubeng -V")
```
Comment (chenrui333) [inline Formula/m/mubeng.rb:16]:
```suggestion
    output = shell_output("#{bin}/mubeng 2>&1", 1)
    assert_match "no proxy file provided", output

    assert_match version.to_s, shell_output("#{bin}/mubeng --version", 1)
```

**Comment by dwisiswant0** [comment]
> @dwisiswant0 can you update your PR to allow push access?

Sure thing, done.

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@dwisiswant0, thanks for your first contribution to homebrew-core! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request merged**